core/src/main/java/org/infinispan/commands/CancelCommand.java

```diff
@@ -20,13 +20,12 @@
  * @author Vladimir Blagojevic
  * @since 5.2
  */
-public class CancelCommand extends BaseRpcCommand implements InitializableCommand {
+public class CancelCommand extends BaseRpcCommand {
```

Comment:
can we remove this command?
Also,  `CancellableCommand` and `CancellationService`? I think it was used by the old distributed executor.

Reply:
@wburns any thoughts?

Reply:
They were used by the old DistributedExecutor. I don't think they are used by anything else now.

Reply:
I've created https://issues.redhat.com/browse/ISPN-11269 and I'll add a commit to this PR